src/Sql/Sql.Test/ScenarioTests/ManagedInstanceProtectorTests.ps1

```diff
@@ -0,0 +1,269 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+# A managed instance can be provisioned using instructions here https://docs.microsoft.com/en-us/azure/sql-database/sql-database-managed-instance-get-started
```

Comment:
Please create MI in test setup. This is already done by other MI cmdlet tests. The trick is to use constant subnet name so that you only have to create the subnet & virtual cluster once (that first time is really slow)

Reply:
addressed